### ISSUE TITLE
Metrics for JS event bus

### ISSUE THREAD

**Issue by macobo** [PostHog/plugin-server#354]
Under https://github.com/PostHog/plugin-server/issues/264 Marius wrote:

>because there are too many other unresolved promises jamming the JS event bus, but the single threaded event loop is stuck behind something sync and stupid, like running getPiscinaStats 3x per event.

If this is a factor in under-load operations of plugin-server, we should also have internal metrics for it to point to when there are issues.

**Comment by mariusandra** [comment]
Absolutely. The simplest canary to detect trouble could be something like this:

```js
setInterval(() => {
    const time = new Date().valueOf()
    setTimeout(() => {
        const lag = new Date().valueOf() - time - 1 // remove that 1ms we actually waited
        logToStatsd('plugin server health', time > 1000 ? 'onfire' : 'thisisfine')
    }, 1)
}, 10000)
```
